certif/tests/unit/adapters/session_test.js

```diff
@@ -164,4 +164,22 @@ module('Unit | Adapter | session', function (hooks) {
       });
     });
   });
+
+  module('#dismissLiveAlert', () => {
+    test('should build save url from session id and candidate id', async function (assert) {
+      // given
+      adapter.ajax = sinon.stub();
+      const sessionId = 123;
+      const candidateId = 456;
+      const payload = { data: { data: { attributes: { 'user-id': candidateId } } } };
+
+      // when
+      await adapter.dismissLiveAlert(sessionId, candidateId);
+
+      // then
+      assert.ok(
+        adapter.ajax.calledWith(`http://localhost:3000/api/sessions/${sessionId}/dismiss-live-alert`, 'PATCH', payload),
```

Comment:
Proposition : éviter de rendre fix la notion de localhost et encore plus de port dans un test

```suggestion
        adapter.ajax.calledWith(`${ENV.APP.API_HOST}/api/sessions/${sessionId}/dismiss-live-alert`, 'PATCH', payload),
```